CMakeSettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
```

Comment:
Do we need this file?

Reply:
It is considered a good gesture to somehow comment on the resolved code review issues.